.github/workflows/test-and-lint.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Install dependencies
         run: npm install
       - name: Compile TypeScript
-        run: tsc
+        run: npx tsc
```

Comment:
The npm test script actually includes compiling TypeScript already, so I think this step is not necessary.

Reply:
I agree 👍  I wasn't sure exactly why this separate `tsc` was called out in the first place, other than a sanity check, but I'll go ahead and remove this step.

Reply:
Yeah, just saw that it was already unnecessary on Travis. Just an oversight when we were working on the whole TS port I guess.